app/assets/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,44 @@
+.card {
+  border-radius: 25px;
+  border-width: 3px;
+  border-color: #fff;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  border: 1px solid blue;
+}
+.card-body p {
+  margin-bottom: -20px;
+}
+.justify-content-between {
```

Comment:
dont override the bootstrap utility class. this class is meant to set justify-content: space-between only.

use something else to set the css properties required